queries/svelte/highlights.scm

```diff
@@ -8,6 +8,17 @@
   (as)
 ] @keyword
 
+((special_block_keyword) @keyword.coroutine
+  (#match? @keyword.coroutine "^(await|then)$"))
+
+(then) @keyword.coroutine
```

Comment:
I don't think we should include .then since it's just a function call, not a special syntax

Reply:
Which then (match or rule)? Or both?

Reply:
both, it should be the same colour as a regular function
nothing special to it IMO

Reply:
Sure, thanks for your input!